docs/language-basics/namespaces.md

```diff
@@ -3,33 +3,33 @@ id: namespaces
 title: Namespaces
 ---
 
-# Namespaces
+# Understanding Namespaces in TypeSpec
```

Comment:
I don't know if I like the change of the titles in the language basics.

The goal is to show all teh basic elements you can define in typespec. I think it was good before to just talk about that exact type. Here it will make the search much less clear as yo have a lot more text to see when search for "Namespace"

Reply:
Yeah, strongly in favor of keeping the existing titles.